applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/prj_release.conf

```diff
@@ -121,48 +123,16 @@ CONFIG_NVS_LOOKUP_CACHE=y
 # Average Thingy:53 current consumption
 CONFIG_AVERAGE_CURRENT_CONSUMPTION=10000
 
-#########################################################
-# Bootloader and Device Firmware Upgrade related settings
-
-CONFIG_BOOTLOADER_MCUBOOT=y
+# Bootloader and MCUmgr (OTA DFU) Configuration
 CONFIG_MCUBOOT_IMAGE_VERSION="2.0.0+0"
-CONFIG_UPDATEABLE_IMAGE_NUMBER=2
-
-# QSPI configuration
-CONFIG_NORDIC_QSPI_NOR=y
-CONFIG_NORDIC_QSPI_NOR_FLASH_LAYOUT_PAGE_SIZE=4096
-CONFIG_NORDIC_QSPI_NOR_STACK_WRITE_BUFFER_SIZE=16
 
-# MCU Manager and SMP configuration
-CONFIG_MCUMGR=y
-CONFIG_MCUMGR_CMD_IMG_MGMT=y
-CONFIG_MCUMGR_CMD_OS_MGMT=y
-CONFIG_MCUMGR_SMP_BT=y
-CONFIG_MCUMGR_SMP_BT_AUTHEN=n
-CONFIG_MCUMGR_BUF_COUNT=6
```

Comment:
We should not remove the buf count config here

Reply:
thanks, I will fix this in the next batch